scripts/loader

```diff
@@ -79,6 +79,7 @@ while [ $j -le 3 ] && [ $sigterm_received -eq 0 ];  do
   101)
     echo "Rebooting host"
     sudo reboot
+    exit 101
```

Comment:
This will cause systemd to restart us which may or may not happen before the reboot. I think you need to just pause here

Reply:
After calling reboot, systemd will try to stop the service, so I think here systemd won't restart it after exit.
But how about just `exit 0` here, because we finished successfully as far as systemd is concerned?

Reply:
That's probably OK, yes